_content/gateways/cisco/setup.md

```diff
@@ -153,6 +153,8 @@ gps ubx enable # Enable GPS
 exit # Exit global configuration mode
 ```
 
+> 📜 This command may return the message `packet-forwarder firmware is not installed` this is totally normal and should be ignored.
```

Comment:
I wouldn't consider this totally normal. Please reword to `This message can be ignored`.

Reply:
as the problem is only for private networks i didn't add it...